extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,963 @@
+
+#### Level Offsets
+
+In the content and metadata objects, a level offset and a default value can be specified to implicitly apply the default value to each tile in all levels before the specified level offset.
+
+#### Level Offset Fill
+
+The level offset fill describes the state of all tiles in levels before the level offset.
+
+### Bitstream Storage
+
+Bitstreams are stored in `bufferView`s, which are typed views into a binary buffer. The value of any `bufferView` property used in this extension is an index into the `bufferViews` array of the `3DTILES_binary_buffers` extension.
+
+### Bitstream Layout
+
+Each bitstream encodes tile state information using an adaptive linear quadtree or octree (depending on the `tilingScheme`) representation of the hierarchy. Tile states are packed in increasing order of level and Morton index of tiles within that level. Only tiles that subdivide internally will have information about their children present at the next level. This allows for a space efficient representation of sparse hierarchies.
+
+*The following section is non-normative.*
+
+The following example illustrates the usage of these buffers in a sparse quadtree that extends to 2 levels. 
+
+```json
+{
+    "tilingScheme": "quadtree",
+    "subdivision": {
+        "bufferView": 0,
+        "maximumLevel": 2
+    },
+    "content": {
+        "levelOffset": 2,
+        "levelOffsetFill": 1,
+        "bufferView": 1
+    },
+    "metadata": {
+        "levelOffset": 2,
+        "levelOffsetFill": 0,
+        "bufferView": 0
+    }
+}
+```
+
+Only the information with bold text is present in the bitstream.
+
+![States](figures/states.png)
+
+## Properties Reference
+
+---------------------------------------
+### 3DTILES_implicit_tiling Tileset JSON extension
+
+Specifies the Tileset JSON properties for the 3DTILES_implicit_tiling.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**boundingVolume**|`object`|A bounding volume that encloses the tileset.|☑️ Yes|
+|**tilingScheme**|`string`|A string describing the tiling scheme used within the tileset.|☑️ Yes|
+|**refine**|`string`|A string to indicate if additive or replacement refinement is used when traversing the tileset for rendering. This property will apply to all available levels in the tileset.|☑️ Yes|
+|**subdivision**|`object`|An object containing high level information about the subdivision buffer|☑️ Yes|
+|**content**|`object`|An object containing high level information about the content buffer. This may be omitted if no tiles in the tileset contain content.|No|
+|**metadata**|`object`|An object containing high level information about the metadata buffer. This may be omitted if no tiles in the tileset contain metadata.|No|
+|**contentExtension**|`string`|The extension applied to each tile in the tileset.|No|
+|**tilesetExtension**|`string`|The extension applied to each implicit external tileset in the tileset.|No|
```

Comment:
`geometricError` should be in the extension's property reference as a way to store the root's geometric error. The root's geometric error differs from the top-level tileset geometric error. It should be a required property.

There should also be a note about how geometric error gets computed for child tiles. As written now the geometric error would divide by two for each level. Longer term geometric error could be encoded in a tile state buffer.

Reply:
Updated to include `geometricError` in property reference and in samples.

Regarding geometric error bitstream, would adding a custom multiplier suffice? Keep in mind that the geometric error would be reset in an external tileset.